### PR TITLE
Add additional links to Learn Chef Rally content

### PR DESCRIPTION
**Pull request by tpetchel**
This change adds links to specific Learn Chef content to help users learn about certain topics.

I also fixed the "under construction" page. It was not rendering correctly.
I also corrected a few caplitalization issues and edited a few sentences for clarity.

### CODE REVIEW

File: www/source/under_construction.html.slim [hunk 1]
```diff
@@ -1,7 +1,11 @@
...
-  a.button href="https://github.com/chef/inspec/issues" target="_blank" Open an issue
+.row.margin-both
+  .columns.medium-12
+    .under-construction
+      h1 Almost there...
+      p
+        | This page is under construction. Let us know if you discovered a broken link and we'll be happy to fix it!
+      nav.ul
+        li
+          a.button href="http://slack.habitat.sh" target="_blank" Reach out on Slack
```
Comment (chris-rock) [inline www/source/under_construction.html.slim]: While we are at it, we need to use `http://community-slack.chef.io/` here

**Review by chris-rock — CHANGES_REQUESTED** [review]
Great improvement @tpetchel I found a small thing we need to update

**Comment by tpetchel** [comment]
@chris-rock - sorry I missed your suggestion. Great catch! I patched that up and merged against the latest `master`.

**Review by miah — APPROVED** [review]
This looks good. I think after the conflict is fixed we can merge.

**Comment by jquick** [comment]
Hey @tpetchel - Do you mind rebasing this vs master? We had a few doc changes that went in.

**Review by jquick — APPROVED** [review]
Thanks @tpetchel !

**Review by chris-rock — APPROVED** [review]
Thank you @tpetchel

**Pull request merged**